addons/account/models/account_move.py

```diff
@@ -1344,7 +1344,9 @@ def _check_fiscalyear_lock_date(self):
 
     @api.multi
     def _check_tax_lock_date(self):
```

Comment:
Is it called from a constraint somewhere?

Reply:
in post